specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-10-01/bms.json

```diff
@@ -0,0 +1,313 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-10-01",
+    "title": "RecoveryServicesBackupClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupEncryptionConfigs/backupResourceEncryptionConfig": {
+      "get": {
+        "tags": [
+          "BackupResourceEncryptionConfigs"
+        ],
+        "description": "Fetches Vault Encryption config.",
+        "operationId": "BackupResourceEncryptionConfigs_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/BackupResourceEncryptionConfigResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/NewErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Vault Encryption Configuration": {
+            "$ref": "./examples/BackupResourceEncryptionConfig_Get.json"
+          }
+        }
+      },
+      "put": {
```

Comment:
>put [](start = 7, length = 3)

I see that there's no delete. DOes this mean that the resource always exists and can only be updated?

Reply:
That is right. This can only be updated.